sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql

```diff
@@ -96,6 +96,70 @@ combined AS (
     -- For firefox 116+ use firefox_desktop.quick_suggest instead
     -- https://bugzilla.mozilla.org/show_bug.cgi?id=1836283
     SAFE_CAST(metadata.user_agent.version AS INT64) < 116
+  UNION ALL
+  -- Suggest Android
+  SELECT
+    metrics.uuid.fx_suggest_context_id AS context_id,
+    DATE(submission_timestamp) AS submission_date,
+    'suggest' AS source,
+    IF(
+      metrics.string.fx_suggest_ping_type = "fxsuggest-click",
+      "click",
+      "impression"
+    ) AS event_type,
+    'phone' AS form_factor,
+    normalized_country_code AS country,
+    metadata.geo.subdivision1 AS subdivision1,
+    metrics.string.fx_suggest_advertiser AS advertiser,
+    client_info.app_channel AS release_channel,
+    metrics.quantity.fx_suggest_position AS position,
+    -- Only remote settings is in use on mobile
+    'remote settings' AS provider,
+    -- Only standard suggestions are in use on mobile
+    'firefox-suggest' AS match_type,
+    SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
+    -- This is the opt-in for Merino, not in use on mobile
+    FALSE AS suggest_data_sharing_enabled,
```

Comment:
Maybe this should actually be NULL, since it's an option that doesn't exist at all on mobile

Reply:
I'm good with either but lets make sure the schema supports a nullable boolean for that column